core/templates/dev/head/player/exploration_player.html

```diff
@@ -105,6 +65,100 @@
     <feedback-popup></feedback-popup>
   </script>
 
+  <script type="text/ng-template" id="popover/informationCard">
+    <md-card class="md-padding">
+      <div ng-style="infoCardBackgroundImage" class="oppia-info-card-logo-thumbnail">
+        <h2 class="oppia-info-card-title"><[explorationCardTitle]></h2>
+      </div>
+
+      <div class="oppia-info-card-content">
+        <p ng-if="explorationDescription">
+          <[explorationDescription | truncateAndCapitalize: 95]>
+        </p>
+        <p ng-if="!explorationDescription">No objective specified.</p>
+        <ul layout="row" class="card-metrics" layout-align="space-between center">
+          <li>
+            <span class="fa fa-star fa-lg"></span><[explorationRatings]></li>
+          <li>
+            <span class="fa fa-eye fa-lg"></span><[viewsCount | summarizeNumber]></li>
+          <li>
+            <span class="fa fa-clock-o fa-lg"></span><[lastModifiedDate | date: 'MM/dd/yyyy']></li>
+          <ul layout="row" layout-align="space-between center" class="oppia-info-card-exploration-contributors-profile">
+            <img src="/third_party/static/material-design-icons-1.0.1/ic_group_black_48dp.png">
+            <li ng-repeat="contributor in explorationContributors| limitTo: 2"
+              tooltip="<[contributor]>" tooltip-placement="top">
+              <img class="img-circle" ng-src="/images/general/user_mint_48px.png" alt=""></li>
```

Comment:
Bring `</li>` down to next line. (Only keep it on the same line if the `<li>` is on that line too, otherwise the indentation looks weird.)